fxa-auth-db-server/docs/API.md

```diff
@@ -62,34 +63,38 @@ The following datatypes are used throughout this document:
 * Devices:
     * createDevice              : `PUT /account/:id/device/:deviceId`
     * updateDevice              : `POST /account/:id/device/:deviceId/update`
-    * deleteDevice              : `DEL /account/:id/device/:deviceId`
+    * deleteDevice              : `DELETE /account/:id/device/:deviceId`
 * Session Tokens:
     * sessionToken              : `GET /sessionToken/:id`
     * sessionTokenWithVerificationStatus : `GET /sessionToken/:id/verified`
     * sessionWithDevice         : `GET /sessionToken/:id/device`
-    * deleteSessionToken        : `DEL /sessionToken/:id`
+    * deleteSessionToken        : `DELETE /sessionToken/:id`
     * createSessionToken        : `PUT /sessionToken/:id`
     * updateSessionToken        : `POST /sessionToken/:id/update`
 * Account Reset Tokens:
     * accountResetToken         : `GET /accountResetToken/:id`
-    * deleteAccountResetToken   : `DEL /accountResetToken/:id`
+    * deleteAccountResetToken   : `DELETE /accountResetToken/:id`
 * Key Fetch Tokens:
     * keyFetchToken             : `GET /keyFetchToken/:id`
     * keyFetchTokenWithVerificationStatus : `GET /keyFetchToken/:id/verified`
-    * deleteKeyFetchToken       : `DEL /keyFetchToken/:id`
+    * deleteKeyFetchToken       : `DELETE /keyFetchToken/:id`
     * createKeyFetchToken       : `PUT /keyFetchToken/:id`
 * Password Change Tokens:
     * passwordChangeToken       : `GET /passwordChangeToken/:id`
-    * deletePasswordChangeToken : `DEL /passwordChangeToken/:id`
+    * deletePasswordChangeToken : `DELETE /passwordChangeToken/:id`
     * createPasswordChangeToken : `PUT /passwordChangeToken/:id`
 * Password Forgot Tokens:
     * passwordForgotToken       : `GET /passwordForgotToken/:id`
-    * deletePasswordForgotToken : `DEL /passwordForgotToken/:id`
+    * deletePasswordForgotToken : `DELETE /passwordForgotToken/:id`
     * createPasswordForgotToken : `PUT /passwordForgotToken/:id`
     * updatePasswordForgotToken : `POST /passwordForgotToken/:id/update`
     * forgotPasswordVerified    : `POST /passwordForgotToken/:id/verified`
 * Unverified tokens:
     * verifyTokens              : `POST /tokens/:tokenVerificationId/verify`
+* Sign-in codes
+    * createSigninCode          : `PUT /signinCodes/:code`
+    * consumeSigninCode         : `DELETE /signinCodes/:code`
```

Comment:
I think the other alternative is `POST`. If we consider that consuming the token is kind of an update to the token then this could potentially make sense? I don't have a strong feeling either way.